nodes.go

```diff
@@ -576,11 +576,20 @@ func setField(tokens []lexer.Token, strct reflect.Value, field structLexerField,
 	}
 
 	if f.Kind() == reflect.Slice {
-		fieldValue, err = conform(f.Type().Elem(), fieldValue)
-		if err != nil {
-			return err
+		if d, ok := reflect.New(f.Type().Elem()).Interface().(Capture); ok {
```

Comment:
This will always create a new element, even if the resulting value doesn't implement `Capture`. Please refactor to avoid this, perhaps using `reflect.Type.Implements`.

Reply:
right, thanks! fixed now